Scripts/replace_nt.py

```
@@ -0,0 +1,35 @@
+import re
+import sys
+from pathlib import Path
+
+REPLACEMENTS = {
+    r"\bNanotrasen\b": "Qillu",
+    r"\bNT\b": "Qillu",
+    r"\bНТ\b": "Qillu",
+    r"\bНанотрейзен\b": "Qillu",
+}
+
+def process_ftl_files(locale_dir: Path):
```

Ruff (0.11.9)

Missing return type annotation for public function `process_ftl_files`

Add return type annotation: `None`

(ANN201)